### DISCUSSION TITLE
Question regarding spritemaps

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1747] [Q&A] [answered]
While strolling through week 7's files, i found these
![Screenshot 2021-09-12 4 52 54 AM](https://user-images.githubusercontent.com/86753001/132983168-2edb448f-a6ce-464a-8eba-cb028d8e4f7e.png)
And inside them gives stuff like this (minus icons and storymenu folder)
![Screenshot 2021-09-12 4 53 06 AM](https://user-images.githubusercontent.com/86753001/132983183-bd3ded99-204e-4053-a1e0-653388930bc2.png)

What are their purpose? And have you considered using them instead of video mp4s

**Comment by catte-cloud** [comment]
Pretty sure its just the new way of loading sprites? I think instead of having giant chonky ass animation XML and PNG files, they instead load it through a json animation.

**Reply by catte-cloud** [reply]
might b wrong though dont quote me lol

**Comment by Cheemsandfriends** [comment, marked as answer]
They are cute texture atlases
Like, there's like a main animation that does the shit and then it exports limb by limb the exported symbol
Adding all the symbols used in that symbol and blablabla

**Reply by Cheemsandfriends** [reply]
I know that cos I made a lil cute repo that parses it
hehehe
